boards/riscv/tlsr9518adk80d/tlsr9518adk80d_pinctrl.dtsi

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2022 Telink Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/b91-pinctrl.h>
+
+&pinctrl {
+	/* Enable PINCTRL driver */
+	status = "okay";
```

Comment:
note: if all your boards depend in practice on `pinctrl` just enable this node at the SoC dts file.

Reply:
Done